src/Reducer/filterReducer.js

```diff
@@ -0,0 +1,13 @@
+import { initialFilterState } from "../Utils/constants";
+
+export const filterReducerFtn = (state , action) => {
+    switch(action.type) {
+        case "CATEGORY" :
```

Comment:
Use constants to avoid string typo bugs

Reply:
yes Thankyou for this suggestion.Will use constants from now on